src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1272,7 +1274,7 @@ class FrontendGenImpl {
     std::string comma = std::string("");
 
     TypeSwitch<Type>(argType)
-        .Case<mlir::onnxmlir::SeqType>([&](mlir::onnxmlir::SeqType seqTy) {
+        .Case<mlir::SeqType>([&](mlir::SeqType seqTy) {
```

Comment:
@etiotto Previously, I put only types (StringType and SeqType) in onnxmlir space. Since I am using table gen to define types now, I can not do that. So I use mlir name space for types as well as Ops temporarily. 
With table gen tool, we can put all the definitions of types and ops in ::mlir::onnx with one line of code. But our existing code are using the namespace explicitly with Ops (like mlir::ONNXConstantOp), it is a tedious but huge change. My plan is to leave that change to another PR.

Reply:
That is fine. We agree that the ONNX and Krnl types eventually should belong to in their appropriate namespace inside mlir::,  I totally understand that we do this in steps.